framework/tests/value/ValueCast.java

```diff
@@ -0,0 +1,11 @@
+// Test case for issue 1264: https://github.com/typetools/checker-framework/issues/1264
```

Comment:
I can see why you view them as separate issues. I will create another issue for this test case and change this reference.